processor/src/main/java/org/mapstruct/ap/internal/model/ContainerMappingMethod.java

```diff
@@ -36,11 +36,8 @@
  *
  * @author Filip Hrisafov
  */
-public abstract class ContainerMappingMethod extends MappingMethod {
```

Comment:
Someday.  A new issue is fine, but not urgent. It works as is. That's what I meant with: 

> I think we should park this one.